app/src/main/java/com/example/changhomework/data/local/datasource/LocalPreferencesSignDataSourceImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.changhomework.data.local.datasource
+
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+class LocalPreferencesSignDataSourceImpl @Inject constructor(
+    private val localPreferences:SharedPreferences
+) : LocalPreferencesSignDataSource {
+    override fun getUserName(id: String): String =
+        localPreferences.getString("${id}name","") ?: ""
+
+
+    override fun getUserinfo(id: String): String =
+        localPreferences.getString(id,"") ?: ""
+
+    override fun setUserName(id: String, name: String) {
+        localPreferences.edit().putString("${id}name",name).apply()
```

Comment:
ktx쓰면 좀 더 간결하게 표현 가능
```suggestion
        localPreferences.edit { putString("${id}name",name) }
```

Reply:
이런거 너무 좋구만 
val mbd1217 = android10duk